app/services/qualified_teachers_api_sender.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+class QualifiedTeachersApiSender
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  SUCCESS_CODES = %w[204].freeze
+
+  attribute :participant_outcome_id
+
+  validates :participant_outcome_id, presence: { message: I18n.t("errors.participant_outcomes.missing_participant_outcome_id") }
+  validates :participant_outcome, presence: { message: I18n.t("errors.participant_outcomes.missing_participant_outcome") }
+  validate :not_already_successfully_sent
+  validate :sent_with_trn_not_found
```

Comment:
chatted in person: maybe we can look for `already_successfully_sent_to_api` being false instead?